### PR TITLE
Change backslashes in include statements to forward slashes

### PR DESCRIPTION
**Pull request by PokeCodec**
Many include statements use forward slashes, while others use backwards
slashes. This is inconsistent formatting. For this reason, I changed the
backward slashes to forward slashes since that is the standard.

### CODE REVIEW

File: src/host/selectionInput.cpp [hunk 1]
```diff
@@ -184,7 +184,7 @@ COORD Selection::WordByWordSelection(const bool fReverse,
 
     // track whether we failed to move during an operation
     // if we failed to move, we hit the end of the buffer and should just highlight to there and be done.
-    bool fMoveSucceeded = false;
+    bool fMoveSucceeded;
 
     // determine if we're highlighting more text or unhighlighting already selected text.
     bool fUnhighlighting;
```
Comment (DHowett) [inline src/host/selectionInput.cpp:190]: please do not introduce unrelated changes. If I recall correctly, we've had this discussion before?

Comment (DHowett) [reply src/host/selectionInput.cpp:190]: It's hard enough to review a pull request that changes 174 files when the _only_ changes are the ones you tell us about. It's impossible to review them when there could be other random changes inside it. :smile:

**Comment by carlos-zamora** [comment]
Curious, is there a way to add this to the code formatter (or checking the code formatting)?

File: src/terminal/adapter/ut_adapter/inputTest.cpp [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 #include <windows.h>
 #include <wextestclass.h>
-#include "..\..\inc\consoletaeftemplates.hpp"
+#include "../../inc/consoletaeftemplates.hpp"
 
-#include "..\..\input\terminalInput.hpp"
+#include "../../input/terminalInput.hpp"
 
 #ifdef BUILD_ONECORE_INTERACTIVITY
 #include "..\..\..\interactivity\inc\VtApiRedirection.hpp"
```
Comment (zadjii-msft) [inline src/terminal/adapter/ut_adapter/inputTest.cpp:13]: Missed one?

File: src/terminal/input/terminalInput.cpp [hunk 3]
```diff
@@ -8,14 +8,14 @@
 #include "strsafe.h"
 
 #define WIL_SUPPORT_BITOPERATION_PASCAL_NAMES
-#include <wil\Common.h>
+#include <wil/Common.h>
 
 #ifdef BUILD_ONECORE_INTERACTIVITY
 #include "..\..\interactivity\inc\VtApiRedirection.hpp"
```
Comment (zadjii-msft) [inline src/terminal/input/terminalInput.cpp:14]: And another here?

**Review by zadjii-msft — APPROVED** [review]

**Comment by zadjii-msft** [comment]
> 
> 
> Curious, is there a way to add this to the code formatter (or checking the code formatting)?

From what I could tell, there wasn't a way to tell `clang-format` to do this unfortunately 😕

**Comment by PokeCodec** [comment]
Ready for review!

**Comment by PokeCodec** [comment]
@DHowett Could you please review when you get the chance?

Thank you!

**Review by DHowett — APPROVED** [review]
Thank you for doing this!
